pyrad/tests/testPacket.py

```diff
@@ -83,6 +85,49 @@ def setUp(self):
             id=0, secret=six.b('secret'),
             authenticator=six.b('01234567890ABCDEF'), dict=self.dict)
 
+    def _create_reply_with_duplicate_attributes(self, request):
```

Comment:
The bug being fixed in this commit occurs when two attributes of the same type do not appear sequentially in the attribute list. By it's nature, the Packet class will never construct a packet with that format (it always groups all the instances of a given attribute together). Therefore the packets needed for testing needed to be built manually.